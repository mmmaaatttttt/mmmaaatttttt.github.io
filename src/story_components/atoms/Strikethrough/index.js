import styled from "styled-components";

const Strikethrough = styled.span`
  text-decoration: line-through;
`

export default Strikethrough;