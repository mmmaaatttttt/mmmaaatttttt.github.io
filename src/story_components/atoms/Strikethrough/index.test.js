import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Strikethrough from ".";

it("renders successfully", () => {
  shallow(<Strikethrough />);
});

it("matches snapshot", () => {
  const wrapper = shallow(
    <Strikethrough>striking out</Strikethrough>
  );
  expect(toJson(wrapper)).toMatchSnapshot();
});
