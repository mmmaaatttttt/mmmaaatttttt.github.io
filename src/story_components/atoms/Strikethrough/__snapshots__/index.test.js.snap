// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
  text-decoration: line-through;
",
        ],
      },
      "displayName": "styled.span",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "span",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
>
  striking out
</StyledComponent>
`;
